galaxy_importer/main.py

```diff
@@ -90,15 +90,18 @@ def call_importer(filepath, cfg):
     namespace, name, version = match.groups()
     filename = collection.CollectionFilename(namespace, name, version)
 
-    with open(filepath, 'rb') as f:
-        try:
-            data = collection.import_collection(f, filename, cfg=cfg)
-        except ImporterError as e:
-            logging.error(f'The import failed for the following reason: {str(e)}')
-            return None
-        except Exception:
-            logging.error('Unexpected error occurred:', exc_info=True)
-            return None
+    try:
+        data = collection.import_collection(
+            file=None,
+            filepath=filepath,
+            filename=filename,
+            cfg=cfg)
+    except ImporterError as e:
+        logging.error(f'The import failed for the following reason: {str(e)}')
```

Comment:
Suggestion: Include the filename/filepath in the log message explicitly (in addition to anything that may be included in the str(e))'

As far as I can tell, the places where ImprotError is raised in the extract path show the str() of either FileNotFoundError or SubprocessError, which should usually include the name of the file being included.  But if there is an ImportError raised elsewhere, I would suggest going ahead and logging the filename/filepath contextual info as well. Some of the places ImportError derived classes are raised don't include much info.